jboss-eap7.3-clustered-multivm-rhel8/README.md

```diff
@@ -165,6 +166,10 @@ This section provides information on troubleshooting some of the issues you migh
 
     `more jbosseap.install.log`
 
+If you select BYOS as RHEL OS License Type make sure to complete the prerequisite steps before starting your deployment.  To provision the RHEL-BYOS VM in your subscription, you must enable it for Cloud Access from the Red Hat Customer Portal and activate Red Hat Gold Images for your subscription. You can enable you subscription for Cloud Access by following the instructions in this Red Hat KB article: [Enabling and Maintaining Subscriptions for Cloud Access](https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/red_hat_cloud_access_reference_guide/con-enable-subs) and activate the Red Hat Gold Images by following the instructions defined in this Red Hat KB article for [enabling Azure Access](https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/red_hat_cloud_access_reference_guide/using_red_hat_gold_images#con-azure-access). If this is not done you will encounter errors with validating the template due to 'marketplace purchase eligibility failed'.
```

Comment:
We have completely changed the structure of the “Troubleshooting” section in the new Draft in https://github.com/SpektraSystems/redhat-mw-cloud-quickstart/blob/master/jboss-eap-standalone-rhel7/README.md. Please review and let us know the feedback, so that we can replicate the same format here.